go.mod

```diff
@@ -1,17 +1,17 @@
 module github.com/containers/skopeo
 
 // Minimum required golang version
-go 1.21
+go 1.21.0
 
 // Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates
-toolchain go1.21.0
+toolchain go1.21.9
```

Comment:
I think we want to stay on 1.21.0 here, so that downstream builders are not constrained (and then `go mod tidy` removes the line; compare https://github.com/containers/common/pull/2083 ).

Reply:
Nice catch, I missed that.  That bump was all auto done for me.